Add a "good to know" note to JS Basic Options docs (#9445)
getsentry/sentry-docs: docs/platforms/javascript/common/configuration/options.mdx

The list of common options across SDKs. These work more or less the same in all SDKs, but some subtle differences will exist to better support the platform. Options that can be read from an environment variable (`SENTRY_DSN`, `SENTRY_ENVIRONMENT`, `SENTRY_RELEASE`) are read automatically.

<Note>

> If you are using a framework or runtime that requires a prefix to expose environment variables to the client, these options will not be read into the client side automatically and must be set via the `init()` function.

Reviewer: ```suggestion
If you are using a framework or runtime that requires a prefix to expose environment variables to the client, you must set these options in the `init()` function. These options will not be read into the client side automatically.
```

> (For example, `NEXT_PUBLIC_` in [Next.js](https://nextjs.org/docs/app/building-your-application/configuring/environment-variables) or `VITE_` in [Vite](https://vitejs.dev/guide/env-and-mode.html#env-files).)

Reviewer: ```suggestion
This applies to `NEXT_PUBLIC_` in [Next.js](https://nextjs.org/docs/app/building-your-application/configuring/environment-variables) and `VITE_` in [Vite](https://vitejs.dev/guide/env-and-mode.html#env-files), among others.
```
